contracts/services/InstanceOperatorService.sol

```diff
@@ -93,6 +93,14 @@ contract InstanceOperatorService is
         _registry.deregisterInRelease(_release, _contractName);
     }
 
+    function contracts() external view override returns (uint256 numberOfContracts) {
```

Comment:
these functions belong into InstanceService, not instanceOperatorService. InstanceOperatorService should only contain state changing functions that are linked to persmissions from the InstanceOperator